### ISSUE TITLE
Shortcuts!

### ISSUE THREAD

**Issue by avm-xandry** [YoRyan/nuxhash#32]
I think that it would be nice if nuxhash-gui can be created shortcuts (.desktop file) which will be work with different DE such Gnome-shell, KDE, etc...
At the moment each user need to create shell-script himself, but it's inconvenient.

**Comment by YoRyan** [comment]
Good idea. Will investigate if there's a neat way for Python packages to do this. Worst-case scenario, I include a suggested `.desktop`.
